src/ensembl/production/metadata/api/models/genome.py

```diff
@@ -36,6 +36,9 @@ class Genome(Base):
     # organism_id to organism
     organism = relationship("Organism", back_populates="genomes")
 
+    def __repr__(self):
```

Comment:
If you are going to add the repr, please be standard between all model classes. These are of a different format than assembly.

Reply:
I removed them for now